_posts/2019-01-19-spidernet-romania.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Spidernet Romania | WoW Private Server
+website: https://www.spidernetromania.net
+author: Alex
+image:https://www.spidernetromania.net/application/images/misc/banner.jpg (suggested dimensions width: 468px  height: 60px )

```

Comment:
only put the image link, remove "  (suggested dimensions width: 468px  height: 60px )"